dp/protos/BUILD.bazel

```diff
@@ -28,3 +28,21 @@ python_grpc_library(
     name = "enodebd_dp_python_grpc",
     protos = [":enodebd_dp_proto"],
 )
+
+proto_library(
+    name = "cbsd_proto",
+    srcs = ["cbsd.proto"],
+    deps = [
+        "@com_google_protobuf//:wrappers_proto",
```

Comment:
Nitpicking: could we format this as `    deps = ["@com_google_protobuf//:wrappers_proto"],`

Reply:
Done.